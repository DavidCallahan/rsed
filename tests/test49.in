{
  "array_random500k" : 7496107666.597366,
  "foldSparse1M" : 60573132666.59737,
  "foldDense100K" : 6600725666.597366,
  "array_random1m" : 14980436666.597366,
  "Yifei_1M_MaxId_1M" : 1769071666.5973663,
  "array_random50k_logestic" : 1004495666.5973663,
  "array_seq1m" : 781083666.5973663,
  "Yifei_1M_MaxId_100M" : 262146395666.59735,
  "array_random100k" : 1475617666.5973663,
  "array_cpy1m" : 1048667666.5973663,
  "array_random30k" : 428868666.59736633,
  "array_seq200k_logistic" : 3439864666.5973663,
  "Yifei_1M_MaxId_10M" : 25012872666.597366,
  "foldSparse100K" : 6154303666.597366,
  "array_random50k" : 715234666.5973663,
  "foldDense1M" : 50411859666.59737,
  "array_seq1m_logistic" : 16567729666.597366,
  "array_random100k_logestic" : 2016218666.5973663,
  "array_random30k_logestic" : 601046666.5973663,
  "array_random50k_plus" : 710456666.5973663,
  "array_random30k_plus" : 426067666.59736633,
  "Yifei2" : 3605566666.5973663,
  "array_random1m_sort" : 15612375666.597366,
  "array_seq200k" : 361703666.59736633,
  "array_seq500k" : 504536666.59736633
}
