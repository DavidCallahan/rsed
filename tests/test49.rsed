skip past required "^\{$"
common = '$run_id,"$REPO","$REVISION","$TARGET",'
template = '"$benchmark",$instance,"success","$time","ns"'r
foreach to required "^\}$"
   required '"(.*)" : ([0-9]+\.[0-9]+)'
   benchmark =$1
   time = $2
   print $common expand($template)
end
