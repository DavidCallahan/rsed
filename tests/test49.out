,"","","","array_random500k",,"success","7496107666.597366","ns"
,"","","","foldSparse1M",,"success","60573132666.59737","ns"
,"","","","foldDense100K",,"success","6600725666.597366","ns"
,"","","","array_random1m",,"success","14980436666.597366","ns"
,"","","","Yifei_1M_MaxId_1M",,"success","1769071666.5973663","ns"
,"","","","array_random50k_logestic",,"success","1004495666.5973663","ns"
,"","","","array_seq1m",,"success","781083666.5973663","ns"
,"","","","Yifei_1M_MaxId_100M",,"success","262146395666.59735","ns"
,"","","","array_random100k",,"success","1475617666.5973663","ns"
,"","","","array_cpy1m",,"success","1048667666.5973663","ns"
,"","","","array_random30k",,"success","428868666.59736633","ns"
,"","","","array_seq200k_logistic",,"success","3439864666.5973663","ns"
,"","","","Yifei_1M_MaxId_10M",,"success","25012872666.597366","ns"
,"","","","foldSparse100K",,"success","6154303666.597366","ns"
,"","","","array_random50k",,"success","715234666.5973663","ns"
,"","","","foldDense1M",,"success","50411859666.59737","ns"
,"","","","array_seq1m_logistic",,"success","16567729666.597366","ns"
,"","","","array_random100k_logestic",,"success","2016218666.5973663","ns"
,"","","","array_random30k_logestic",,"success","601046666.5973663","ns"
,"","","","array_random50k_plus",,"success","710456666.5973663","ns"
,"","","","array_random30k_plus",,"success","426067666.59736633","ns"
,"","","","Yifei2",,"success","3605566666.5973663","ns"
,"","","","array_random1m_sort",,"success","15612375666.597366","ns"
,"","","","array_seq200k",,"success","361703666.59736633","ns"
,"","","","array_seq500k",,"success","504536666.59736633","ns"
